kubric/viewer/blender.py

```diff
@@ -298,6 +299,33 @@ def blender_apply(self, blender_object):
     bpy.ops.object.shade_flat()
 
 
+class MeshChromeMaterial(interface.MeshBasicMaterial, Material):
+  def __init__(self, color_hsv=(1.0,1.0,1.0), roughness=.5):
+    color_rgba = interface.hsv_to_rgba(color_hsv)
+
+    # --- Create node-based material
+    self._blender_material = bpy.data.materials.new('Chrome')
+    self._blender_material.use_nodes = True
+    tree = self._blender_material.node_tree
+
+    # --- Specify nodes
+    LW = tree.nodes.new('ShaderNodeLayerWeight')
+    LW.inputs[0].default_value = 0.7
+    CR = tree.nodes.new('ShaderNodeValToRGB')
+    CR.color_ramp.elements[0].position = 0.9
+    CR.color_ramp.elements[0].color = color_rgba
+    CR.color_ramp.elements[1].position = 1
+    CR.color_ramp.elements[1].color = (0,0,0,1)
+    GLO = tree.nodes.new('ShaderNodeBsdfGlossy')
+    GLO.inputs[1].default_value = roughness
+    GLO.location.x -= 200  # TODO: ???
```

Comment:
Affects only the visual location of the node if using the Blender GUI. Can be nice for manually debugging a `.blend` file, but doesn't affect the rendering in any way.